ddp-workspace/projects/ddp-atcp/src/app/components/participant-list/participant-list.component.ts

```diff
@@ -79,24 +80,32 @@ export class ParticipantListComponent implements OnInit, OnDestroy {
 
   onAddParticipantClick(): void {
     this.disableAddParticipantsButton = true;
+      const payload: AddParticipantPayload = {
+          timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+      };
+
+    const addParticipantObs = this.userProfileServiceAgent.profile
```

Comment:
The flow remains consistent, with a single exception:
Initially, we initiate a request for the participant's profile (Parent One) to establish the governed participant's language.